cmd/kube-ecr-cleanup-controller/main.go

```diff
@@ -33,6 +33,7 @@ func init() {
 	flag.StringVar(&reposStr, "repos", reposStr, "comma-separated list of repository names to watch.")
 	flag.StringVar(&task.AwsRegion, "region", task.AwsRegion, "AWS Region to use when talking to AWS.")
 	flag.BoolVar(&task.DryRun, "dry-run", task.DryRun, "just log, don't delete any images.")
+	flag.StringVar(&registryID, "registry-id", registryID, "Specify a registry account ID. If not specified, uses the account ID of the credentials passed.")
```

Comment:
I suggest using a down-case first letter, so that it follows the convention used for other flags.

```suggestion
	flag.StringVar(&registryID, "registry-id", registryID, "specify a registry account ID. If not specified, uses the account ID of the credentials passed.")
```